### ISSUE TITLE
ensure PD is added to FMID package

### ISSUE THREAD

**Issue by OnnoVdT** [zowe/zowe-install-packaging#1105]
**Describe the bug**

Currently a placeholder PD is added to the smpe package.

**To Reproduce**

**Expected behavior**

ensure PD is added to FMID package

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by stevenhorsman** [comment]
@OnnoVdT - was this completed with PR #1110, or should it be in progress?

**Comment by OnnoVdT** [comment]
no, #1110 is a hack to go straight to the output of what this issue will deliver. With this issue, the build will set aside multiple values that will be picked up and assigned to PD variables, after which the PD is genarated.
